antora-ui-camel/src/css/blog.css

```diff
@@ -99,6 +99,25 @@ article.blog p {
   white-space: pre-wrap;
 }
 
+.blog .post .post-content table {
+  width: 100%;
+  text-align: center;
+  margin-bottom: 1rem;
+}
+
+.blog .post .post-content tr:nth-child(even) {
+  background: #f4f5f7;
```

Comment:
Can we use one of the variables in `vars.css` instead? `--color-smoke-50` looks similar.

Reply:
I have updated the colours from vars.css. Please take a look now!